src/odemis/driver/zeiss.py

```diff
@@ -419,15 +437,26 @@ def Abort(self):
         """
         return self._SendCmd(b'ABO')
 
-    def MoveStage(self, x, y, z):
+    def MoveStage(self, move_list):
         """
-        Absolute move. Non blocking.
+        Absolute move. Non-blocking.
         Use GetStagePosition() to check the move status.
-        x, y, z (floats): absolute target position in mm
+        :param move_list (floats): list with 3 absolute target positions in mm
         """
+        x, y, z = move_list
         c = b'STG %f %f %f' % (x, y, z)
         self._SendCmd(c)
 
+    def MoveStage6(self, move_list):
+        """
+        Absolute move. Non-blocking.
+        Use GetStagePosition6() to check the move status.
+        :param move_list (floats): list with 6 absolute target positions in mm or degree
+        """
+        x, y, z, t, r, m = move_list
+        c = b'C95 %f %f %f %f %f %f' % (x, y, z, t, r, m)
```

Comment:
This is tricky, here one may loose some precision. I don't how important the precision is.

For example, trying following in python:
```
>>> c = b'C95 %f' % (1.123456789)
>>> c
b'C95 1.123457'
```

One workaround is to count the precision for each of `x, y, z, t, r, m` and then use something like follows

```
>>> c = b'C95 %.9f' % (1.123456789)
>>> c
b'C95 1.123456789'
```

Reply:
That's a good remark. I've looked at some logs. STG? (ie, the position as given by the SEM) returns values like that: `45.26179 58.16799 33.24926`. So 5 digits after the decimal point. As it's in mm, that's 10nm, which is probably the best of the stage encoder precision (and a lot better than a move). So the standard `%f`, with 6 decimals should be fine.
For rotations, we don't have logs, but as it's in degrees, and I don't expect a precision more than 0.01°, that should be always fine. 

=> %f is fine, but please add a comment to explain it.